javascript/src/test/components/TutorAssignment/TutorAssignmentTable.test.js

```diff
@@ -40,9 +55,36 @@ describe("CourseForm tests", () => {
     });
 
     const {getByTestId} = render(<TutorAssignmentTable tutorAssignments={tutorAssignments} isInstructor={true}/>);
-    const editButton = getByTestId('edit-button');
+    const editButton = getByTestId('edit-button-1');
     userEvent.click(editButton);
 
     await waitFor(() => expect(pushSpy).toHaveBeenCalledTimes(1));
   });
+
+  test("ascending and descending buttons for course can be pressed", () => {
+
+    const { getByText } = render(<TutorAssignmentTable tutorAssignments={tutorAssignments} isInstructor={true} />);
+    const tutorAssignmentHeader = getByText(/Course/);
+
+    //cycle through both ascending and descending for test coverage
+    userEvent.click(tutorAssignmentHeader);
+    userEvent.click(tutorAssignmentHeader);
+    const descendingOFF = String.fromCharCode(0x25bd);
+    expect(getByText(descendingOFF)).toBeInTheDocument();
+
+  });
+
+  test("ascending and descending buttons for tutor can be pressed", () => {
+
+    const { getByText } = render(<TutorAssignmentTable tutorAssignments={tutorAssignments} isInstructor={true} />);
+    const tutorAssignmentHeader = getByText(/Tutor/);
+
+    //cycle through both ascending and descending for test coverage
+    userEvent.click(tutorAssignmentHeader);
+    userEvent.click(tutorAssignmentHeader);
+    const descendingOFF = String.fromCharCode(0x25bd);
+    expect(getByText(descendingOFF)).toBeInTheDocument();
+
+  });
```

Comment:
These tests might have full test coverage, but you're not really testing anything here. You're checking for the presence of `descendingOFF`, which is always rendered anyway. Maybe you could write a test that tests whether the sort caret is being cycled through?

Reply:
Bryan raises a good point.  But I'm going to go ahead and merge it anyway.     We can come back and try to improve the tests in a later pass.